### PR TITLE
Feature/check human genome file endings

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR fixes:

- Refactor check human genome file endings

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [x] Code review
- [x] New code is executed and covered by tests
- [x] Tests pass

### CODE REVIEW

File: lib/MIP/File_info.pm [hunk 1]
```diff
@@ -24,10 +24,75 @@ BEGIN {
...
+            attribute      => q{build_file},
+            parameter_href => $parameter_href,
+            parameter_name => q{human_genome_reference_file_endings},
+        }
+    );
+
+## File needs to be built before getting contigs
+    return if ($build_status);
+
+    @{ $file_info_href->{contigs} } = get_dict_contigs(
```
Comment (jemten) [inline lib/MIP/File_info.pm]: In the future I think that we should consider renaming this key to {dict_contigs} or similar to separate them from the primary contigs that uses the same key.

File: lib/MIP/File_info.pm [hunk 2]
```diff
@@ -24,10 +24,75 @@ BEGIN {
...
     # Set the version for version checking
-    our $VERSION = 1.00;
+    our $VERSION = 1.01;
 
     # Functions and variables which can be optionally exported
-    our @EXPORT_OK = qw{ set_human_genome_reference_features };
+    our @EXPORT_OK = qw{ set_dict_contigs set_human_genome_reference_features };
+}
+
+sub set_dict_contigs {
```
Comment (jemten) [inline lib/MIP/File_info.pm]: Nice cleanup of sub

File: lib/MIP/Language/Perl.pm [hunk 3]
```diff
@@ -167,13 +167,12 @@ sub perl_nae_oneliners {
...
     my %oneliner = (
-        synonyms_grch37_to_grch38 =>
-          q?\'if($_=~s/^M/chrMT/g) {} elsif ($_=~s/^(.+)/chr$1/g) {} print $_\'?,
-        synonyms_grch38_to_grch37 =>
-          q?\'if($_=~s/^chrMT/M/g) {} elsif ($_=~s/^chr(.+)/$1/g) {} print $_\'?,
-        get_vep_version => q?'if($_=~/ensembl-vep\s+:\s(\d+)/xms) {print $1;}'?,
+        get_dict_contigs          => \&_get_dict_contigs,
+        get_vep_version           => \&_get_vep_version,
+        synonyms_grch37_to_grch38 => \&_synonyms_grch37_to_grch38,
+        synonyms_grch38_to_grch37 => \&_synonyms_grch38_to_grch37,
```
Comment (jemten) [inline lib/MIP/Language/Perl.pm]: ⭐️

File: lib/MIP/File_info.pm (hunk 1 above)
Comment (henrikstranneheim) [reply lib/MIP/File_info.pm]: Agreed. Let's do it another PR to limit any repercussions.

File: lib/MIP/File_info.pm (hunk 2 above)
Comment (henrikstranneheim) [reply lib/MIP/File_info.pm]: Thx!

**Pull request merged**